pages/common/dwebp.md

```diff
@@ -4,6 +4,26 @@
 > Animated WebP files are not supported.
 > More information: <https://developers.google.com/speed/webp/docs/dwebp/>.
 
-- Convert a `webp` file to a `png` file:
+- Convert a `webp` file into a `png` file:
 
 `dwebp {{path/to/input.webp}} -o {{path/to/output.png}}`
+
+- Convert a `webp` file into a specific filetype:
+
+`dwebp {{path/to/input.webp}} -bmp|-tiff|-pam|-ppm|-pgm|-yuv -o {{path/to/output}}`
+
+- Convert a `webp` file, using multi-threading if possible:
+
+`dwebp {{path/to/input.webp}} -o {{path/to/output.png}} -mt`
+
+- Convert a `webp` file and crop and scale the output:
```

Comment:
```suggestion
- Convert a `webp` file, crop, and scale the output:
```

Reply:
I think since `crop` and `scale` both refer to `output`, they should be grouped extra. The sentence sounds a bit weird when just listing everything.

Reply:
What about this?
```suggestion
- Convert a `webp` file, but also crop and scale at the same time:
```